### PR TITLE
[prometheus-blackbox-exporter] allow configurable DNS settings

### PR DESCRIPTION
**Pull request by jstewart612**
Signed-off-by: John Stewart <jstewart@rentpath.com>

### CODE REVIEW

File: charts/prometheus-blackbox-exporter/values.yaml
Comment (torstenwalter) [inline charts/prometheus-blackbox-exporter/values.yaml:163]: I suggest to remove the comment "#". If the values key exits then some IDEs can prove auto-completion for it.

File: charts/prometheus-blackbox-exporter/Chart.yaml
Comment (monotek) [inline charts/prometheus-blackbox-exporter/Chart.yaml:4]: Imho minor version should be changed as new feature is added.

So right version would be version 4.4.0.

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-blackbox-exporter/values.yaml
Comment (jstewart612) [reply charts/prometheus-blackbox-exporter/values.yaml]: Changed and resolved in https://github.com/prometheus-community/helm-charts/pull/42/commits/27e8ee1f9e4ce2320e3a29f55f3e052a95da5075

File: charts/prometheus-blackbox-exporter/Chart.yaml
Comment (jstewart612) [reply charts/prometheus-blackbox-exporter/Chart.yaml]: Changed and resolved in https://github.com/prometheus-community/helm-charts/pull/42/commits/27e8ee1f9e4ce2320e3a29f55f3e052a95da5075

**Review by monotek — APPROVED** [review]

**Comment by jstewart612** [comment]
If I'm thinking this out correctly, @desaintmartin or @gianrubio  are the CODEOWNERS for this chart and thus the only ones that can merge this PR in.

Mentioning so they see it.  Apologies if they see it without being mentioned:  I don't know how your workflow works.

**Comment by desaintmartin** [comment]
Thanks!
/assign

**Comment by torstenwalter** [comment]
@desaintmartin You and @gianrubio should have been invited to the helm-charts-maintainers team. You should have that invite in you inbox. Once you accepta that you should be able to approve this PR.

**Comment by desaintmartin** [comment]
Hm, I don't see it either from my inbox or from the Github notifications @torstenwalter

**Comment by monotek** [comment]
@desaintmartin 
As mentioned in https://github.com/prometheus-community/helm-charts/issues/21#issuecomment-687661218 i could co maintain the chart too, if wanted :)

**Review by desaintmartin — APPROVED** [review]

**Pull request merged**
